### ISSUE TITLE
Triggers Facebook Container

### ISSUE THREAD

**Issue by forteller** [heiseonline/shariff#363]
Thank your for a great Wordpress plugin, I love it!

But I noticed a thing that makes me usure of how well it actually protects my visitors privacy: I have the official Mozilla extension [Facebook Container](https://github.com/mozilla/contain-facebook/). And now it is deactivating the Facebook button from Shariff with this message:

![Screenshot from 2019-05-22 11-59-35](https://user-images.githubusercontent.com/343561/58166162-824da080-7c89-11e9-91b0-7674c01d292a.png)

"Facebook Container has disabled this button and blocked Facebook from tracking your visit to this page."

Why is this? Shariff is just adding a link to Facebook, and never pinging Facebook unless people click the link, right?

Thank you for your time helping a n00b understand :)

**Comment by richard67** [comment]
> Shariff is just adding a link to Facebook, and never pinging Facebook unless people click the link, right?

Yes. It seems Facebook Container just sees that there is a link with target = some facebook URL and blocks it, regardless whether it would track data or not. Seems to be a silly Mozilla extension.

**Comment by richard67** [comment]
You should open an issue in their repository, [https://github.com/mozilla/contain-facebook/]( https://github.com/mozilla/contain-facebook/).

**Comment by forteller** [comment]
Thanks for the reply! I'll do that :)
